scripts/getDependencies.pl

```diff
@@ -129,6 +129,13 @@
 		fname => 'org.eclipse.osgi-3.16.100.jar',
 		sha1 => '7ddb312f386b799d6e004d193a01c50169bf69f3'
 	},
+	jtreg => {
+		url => "https://ci.adoptopenjdk.net/job/dependency_pipeline/lastSuccessfulBuild/artifact/jtreg/$jtregVersion.tar.gz",
+		fname => 'jtreg.tar.gz',
+		shaurl => "https://ci.adoptopenjdk.net/job/dependency_pipeline/lastSuccessfulBuild/artifact/jtreg/$jtregVersion.tar.gz.sha256sum.txt",
```

Comment:
Please explicitly write the jtregVersion. This map should contain both jar info.

Reply:
I updated the getDependencies.pl file with both the jtreg.jar versions.